server/plugin/hooks.go

```diff
@@ -21,18 +21,25 @@ func (p *Plugin) MessageHasBeenPosted(c *plugin.Context, post *model.Post) {
 		return
 	}
 
-	channel, appErr := p.API.GetChannel(post.ChannelId)
-	if appErr != nil {
-		p.API.LogError("Error occurred while fetching channel by ID. ChannelID: %s. Error: %s", post.ChannelId, appErr.Error())
-		return
-	}
+	var channelName string
+	cacheVal, err := p.dmChannelCache.Get(post.ChannelId)
+	if err == nil {
```

Comment:
Are there other errors that `Get` can return that don't originate from a missing cache entry?

Reply:
No, I checked the code for `Get` and there are no other errors if there's no loader func, which is true in our case.